cpp/CMakeLists.txt

```diff
@@ -72,6 +85,18 @@ add_feature_info(
   "Add paths to linker search and installed rpath."
 )
 
+# Control Basix discovery
+option(
+  DOLFINX_BASIX_PYTHON
+  "Enable Basix discovery using Python. Disable if Basix should be found using CMake."
+  ON
+)
+add_feature_info(
+  DOLFINX_BASIX_PYTHON
+  DOLFINX_BASIX_PYTHON
+  "Enable Basix discovery using Python. Disable if Basix should be found using CMake."
```

Comment:
I think this is a bit misleading - Basix is always 'found' by cmake. Python can be used to find where the Basix CMake file is.

Reply:
Will change.